extensions/persistence/org.eclipse.smarthome.persistence.mapdb/src/main/java/org/eclipse/smarthome/persistence/mapdb/internal/MapDbPersistenceService.java

```diff
@@ -53,7 +53,7 @@
  * @author Martin Kühl - Port to Eclipse SmartHome
  */
 @NonNullByDefault
-@Component(service = QueryablePersistenceService.class)
+@Component(service = { PersistenceService.class, QueryablePersistenceService.class })
```

Comment:
since `QueryablePersistenceService` extends `PersistenceService`, doesn't it qualify for service resolution also without the explicit naming?

Reply:
I was surprised to find that it does not.